README.md

```diff
@@ -1,72 +1,41 @@
+You can find the changes for each version under [Releases](https://github.com/AzureAD/azure-activedirectory-library-for-python/releases).
 
-  You'll need a C compiler, libffi + its development headers, and openssl + its development headers.
-  Refer to [cryptography installation](https://cryptography.io/en/latest/installation/)
+> Note: Changes on 'client_id' and 'resource' arguments after 0.1.0
+>
+>The convenient methods in 0.1.0 have been removed, and now your application should provide parameter values to `client_id` and `resource`.
+>
+>2 Reasons:
+>
+>* Each adal client should have an Application ID representing a valid application registered in a tenant. The old methods borrowed the client-id of [azure-cli](https://github.com/Azure/azure-xplat-cli), which is never right. It is simple to register your application and get a client id. You can follow [this article](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-integrating-applications).
+>
+>* The old method defaults the `resource` argument to 'https://management.core.windows.net/', now you can just supply this value explictly. Please note, there are lots of different azure resources you can acquire tokens through adal though, for example, the samples in the repository acquire for the 'graph' resource. Because it is not an appropriate assumption to be made at the library level, we removed the old defaults.
 
-* To install from source:
-
-  Upgrade to the latest pip (8.1.2 as of June 2016).
-  To avoid dealing with compilation errors from cryptography, first run `pip install cryptography` to use statically-linked wheels.
-  Next, run `python setup.py install`
-
-  If you still like to build from source, refer to [cryptography installation](https://cryptography.io/en/latest/installation/).
-  For more context, start with this [stackoverflow thread](http://stackoverflow.com/questions/22073516/failed-to-install-python-cryptography-package-with-pip-and-setup-py).
-
-### Acquire Token with Client Credentials
-
-In order to use this token acquisition method, you need to configure a service principal. Please follow [this walkthrough](https://azure.microsoft.com/en-us/documentation/articles/resource-group-create-service-principal-portal/).
-
-Find the `Main logic` part in the [sample](sample/client_credentials_sample.py#L46-L55).
-
-### Acquire Token with client certificate
-A service principal is also required.
-Find the `Main logic` part in the [sample](sample/certificate_credentials_sample.py#L55-L64).
-
-### Acquire Token with Refresh Token
-Find the `Main logic` part in the [sample](sample/refresh_token_sample.py#L47-L69).
-
-### Acquire Token with device code
-Find the `Main logic` part in the [sample](sample/device_code_sample.py#L49-L54).
-
-### Acquire Token with authorization code
-Find the `Main logic` part in the [sample](sample/website_sample.py#L107-L115) for a complete bare bones web site that makes use of the code below.
-
-## Logging
```

Comment:
I like that we remove this part, because we do have [a similar page in wiki](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki/Logging) now.